ros/src/overhead_cv/overhead_cv/cv_recorder.py

```diff
@@ -0,0 +1,152 @@
+import math
+
+# pylint: disable=E1101
+import cv2
+
+import numpy as np
+
+import rclpy
+import rclpy.exceptions
+import rclpy.executors
+import rclpy.logging
+
+from rclpy.node import Node
+
+from std_msgs.msg import Float32MultiArray
+
+CAMERA_ID = 0
+DELAY = 1
+
+WINDOW_NAME = "OpenCV QR Code"
+
+MAP_WINDOW = "location tracking"
+
+QCD = cv2.QRCodeDetector()
+CAP = cv2.VideoCapture(CAMERA_ID)
+
+
+class CVRecorder(Node):
+    def __init__(self):
+        super().__init__("cv_recorder")
+        self.poses_emit = self.create_publisher(
+            Float32MultiArray, "multi_array_pos", 10
+        )
+
+        self.robot_points = []
+
+    # intended to convert points to width and height of sub image
+    # pylint: disable=R0201
```

Comment:
R0201 is a valid warning. This can be extracted into a function outside of the node, probably in a different file. The name `draw_robot` also doesn't really seem to describe what the function is doing?